### PR TITLE
feat: featured loan test for logged in users in lending home

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- featured loan test for logged in users in lending home

### CODE REVIEW

File: src/pages/LoanFinding/LoanFinding.vue [hunk 1]
```diff
@@ -443,7 +447,7 @@ export default {
 				variables: prefetchedEndingSoonLoanVariables
 			})?.fundraisingLoans?.values[0] ?? { id: 0 };
 
-			this.featuredLoan = cachedEndingSoonLoan;
+			this.featuredLoan = this.isLoggedIn ? this.firstRowLoans[0] : cachedEndingSoonLoan;
```
Comment (dyersituations) [inline src/pages/LoanFinding/LoanFinding.vue:450]: Check array length.

File: src/pages/LoanFinding/LoanFinding.vue [hunk 2]
```diff
@@ -237,10 +237,14 @@ export default {
 				: 'Support a featured borrower with a microloan.';
 		},
 		featuredLoanTitle() {
-			return 'A loan ending soon';
+			return this.isLoggedIn
+				? `Recommended for you, ${this.firstName}`
```
Comment (dyersituations) [inline src/pages/LoanFinding/LoanFinding.vue:241]: The first name text should be extracted into its own element and the class `data-hj-suppress` added to that new element.

**Comment by dyersituations** [comment]
I'm seeing this locally when logged in, and I think there are two issues:

1. The top header probably isn't needed if the message says the user's name (it's not currently -> see #2)
2. I'm not seeing the logged in user message

image

**Comment by dyersituations** [comment]
Disregard my last comment I just deleted, my local auth status was messed up. That being said, I think we should post in our team channel to see if we want to remove the top message. I'll approve, but please ask Nathan.

image

**Comment by roger-in-kiva** [comment]
> Disregard my last comment I just deleted, my local auth status was messed up. That being said, I think we should post in our team channel to see if we want to remove the top message. I'll approve, but please ask Nathan.
> 
> image

Nice call, thanks!

File: src/pages/LoanFinding/LoanFinding.vue [hunk 3]
```diff
@@ -237,10 +238,14 @@ export default {
...
 		featuredLoanTitle() {
-			return 'A loan ending soon';
+			return this.isLoggedIn
+				? 'Recommended for you,'
+				: 'A loan ending soon';
 		},
 		featuredLoanSubtitle() {
-			return 'Make a difference for these borrowers who only have a short time remaining.'; // eslint-disable-line max-len
+			return this.isLoggedIn
+				? 'Make a difference for borrowers who only have a small amount remaining.'
```
Comment (dyersituations) [inline src/pages/LoanFinding/LoanFinding.vue:247]: This message isn't correct, right? We are currently showing first recommended, not with small amount remaining.

Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue:247]: Agreed! But is the copy we have in the mocks, https://www.figma.com/file/6RV1BiJR8s8rqAVlv6dRwl/Lending-Home-V4---Featured-Loan?node-id=3145%3A263&mode=dev

Comment (dyersituations) [reply src/pages/LoanFinding/LoanFinding.vue:247]: Okay I'll approve, but please include that question to Nathan as well in our slack channel.

**Review by dyersituations — APPROVED** [review]

**Review by dyersituations — APPROVED** [review]

**Pull request merged**